app/src/main/java/com/example/party_player/MainActivity.kt

```diff
@@ -236,15 +234,15 @@ class MainActivity : AppCompatActivity() {
 
         searchButton.setOnClickListener {
             //user input song, artist is taken in as a string
-            var songArtist = songTB.text
+            val songArtist = songTB.text
```

Comment:
Here you didn't need "var" because val is immutable, it looks cleaner. to see more about it, check the references.

Reference: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
This makes sense, and is good practice in Kotlin: make things val (final), unless you really plan on changing them at some point.